billing.md

```diff
@@ -138,17 +223,17 @@ To create a subscription, first retrieve an instance of your billable model, whi
 
     $user = User::find(1);
 
-    $user->newSubscription('main', 'premium')->create($token);
+    $user->newSubscription('main', 'premium')->create($paymentMethod);
```

Comment:
What is payment method? String? Object?

Reply:
I've added a link to the "Storing Payment Methods" section for this.

Reply:
@taylorotwell the section below has a link to what this parameter is and how you can retrieve it.